Makefile

```diff
@@ -853,6 +853,11 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint:
+	@echo "Running Markdown linter..."
+	$(NODE) tools/remark-cli/cli.js -q \
+	  doc/api doc/api_assets doc/guides src lib benchmark tools/doc/ tools/icu/
```

Comment:
Hm. There are two lint errors in the whole `doc` dir currently:
```
doc/changelogs/CHANGELOG_V6.md
  1325:95-1325:107  warning  Don’t pad `emphasis` with inner spaces  no-inline-padding  remark-lint

doc/releases.md
       298:1-302:4  warning  Missing code-language flag              fenced-code-flag   remark-lint
```

I don't agree that the commit messages _in the changelog_ should not be touched — those could be actual errors that made changelog look bad. E.g. if a changelog includes `__proto__`, we don't want it to render as `__proto__`, we want to render it as `__proto__` or \_\_proto\_\_ — so amending the changelog to escape those things sounds reasonable to me, and I would want the linter to catch those errors. See commit [a58b48bc3bc](https://github.com/nodejs/node/commit/a58b48bc3bcf43f7090d4cc914606af68fe55815#diff-869344da7e479f3aa0fbe085f3ee688dL715), for example.

So I would prefer if this just replaced `handle__` with either <code>&#96;handle__&#96;</code> or `handle\_\_` in the changelogs (and similarly same for `handle_`).